### ISSUE TITLE
In-File search include images

### ISSUE THREAD

**Issue by Narmjep** [scambier/obsidian-omnisearch#381]
**Is your feature request related to a problem? Please describe.**
I mainly use this plugin with Text Extractor to look for text in images. I assume that searching the entire vault takes more time than using the In-File search. The In-File search however doesn't include the content of the embedded images, forcing me to use vault search.

**Describe the solution you'd like**
Include embedded images in the In-File search (embedded, not linked)

**Describe alternatives you've considered**
Alternatively, create a In-File + Embedded Files search

The implementation should be simple so I hope you consider my request. I would also happily work on this myself if needed. 
Thank you for your efforts; love the plugin!

**Comment by scambier** [comment]
Possible duplicate of #245?

**Comment by Narmjep** [comment]
> Possible duplicate of #245?

I don't think so. #245 is about displaying links that reference a pdf/image from the results.
What I want however is specifically related to In-File search: search in the current file AND in any **images** embedded in that file

**Comment by redstefan1** [comment]
I can second this, I actually stumbled upon this issue exactly because I was trying to use In-File search to search through some big note with lots of images and saw that nothing popped up and thought that it was a bug
